src/main/java/com/shopify/ShopifySdk.java

```diff
@@ -156,15 +161,30 @@
 	private String authorizationToken;
 	private WebTarget webTarget;
 	private String accessToken;
+	private Long minimumRequestRetryRandomDelayMilliseconds;
+	private Long maximumRequestRetryRandomDelayMilliseconds;
+	private Long maximumRequestRetryTimeoutMilliseconds;
 
 	private static final Client CLIENT = buildClient();
 
-	public static interface BuildStep {
+	public static interface OptionalsStep {
```

Comment:
Please add usage and explanation of all these optional fields within the README. 

Actually on second thought, maybe it is better to just add JavaDocs for these settings? Or maybe both?

Use your best judgement but we should document usage somewhere.